targets/arty.py

```diff
@@ -17,6 +17,7 @@ def generate_overlay(tree, overlay):
         set_entry(overlay, bootrom, region, 0x400000)
 
     set_boot_hart(tree, overlay)
+    set_ecc_scrub(tree, overlay)
```

Comment:
no need to add for arty, hifive1, qemu, spike, vc707.

Only add for testbench and vcu118

Reply:
Fixed. Also updated script to generate <metal, ecc scrub> entry in overlay only if beu is present.